src/app/lib/config/services/azeri.js

```diff
@@ -74,6 +74,7 @@ export const service = {
         audioPlayer: 'Audio player',
         videoPlayer: 'Video player',
       },
+      gist: 'Xülasə',
```

Comment:
Got a rectangle of doom here - encoding issue? Xülasə

Reply:
@ryanmccombe Hmm I can't see the Doomtangle, which character position wise is showing that?

Reply:
Can someone clarify the Doomtangle? Sounds intriguing..

Reply:
Oh interesting - only an issue on mobile chrome maybe?

![Screenshot_20210809-134004](https://user-images.githubusercontent.com/8601460/128707641-87f7e2d9-8434-47b3-8279-8ad1df27b384.png)

Reply:
@ryanmccombe That's weird, does it show the rectangle when you view the file through the files changed tab?

Reply:
Yep - this file has numerous rectangles on mobile though so probably just a GitHub or android bug - not a simorgh issue